<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CreateTable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ColumnList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditTableName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Enter table name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Enter database name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditDBName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Data Type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEditColumnName"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Column Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_AddColumn">
      <property name="text">
       <string>Add Column</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>integer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>text</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>numeric</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>real</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>blob</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditDBName</tabstop>
  <tabstop>lineEditTableName</tabstop>
  <tabstop>lineEditColumnName</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_AddColumn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_AddColumn</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>addColmn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_CreateTable</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>createTbl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createDB()</slot>
  <slot>createTbl()</slot>
  <slot>addColmn()</slot>
 </slots>
</ui>
